include/humidity.h

```diff
@@ -1,3 +1,62 @@
-#pragma once
+#ifndef HUMIDITY_H
+#define HUMIDITY_H
 
-int measureHumidity();
\ No newline at end of file
+#include <hih8120.h>
```

Comment:
`lib/drivers/hih8120.h:129:1: error: unknown type name 'uint16_t'
 uint16_t hih8120_getHumidityPercent_x10(void);`
Would suggest adding `#include <stdio.h>` above to include `uint16_t` and other types. It is also required for `src/humidity.c:92:13: note: include '<stdio.h>' or provide a declaration of 'printf'`